contracts/src/v0.8/tests/OCR2DROracleHelper.sol

```diff
@@ -11,9 +11,19 @@ contract OCR2DROracleHelper is OCR2DROracle {
   }
 
   function callReport(bytes calldata report) external {
-    uint32 initialGas;
-    address transmitter;
+    uint32 initialGas = uint32(gasleft());
```

Comment:
Appears to be dangerous to cast like that.
What about https://docs.openzeppelin.com/contracts/3.x/api/utils#SafeCast?
`SafeCase.toUint32(gasLeft())`

Reply:
Good idea. 
Added SafeCast.sol to vendor contracts & used it to cast.